docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -49,6 +49,8 @@ As you make changes, you can rebuild the `docs` module specifically to update th
 ./mvnw -f docs clean install
 ----
 
+yaml gradle dev ..bad stuff to catch vale!
+
```

Comment:
The 2 spelling warnings above for 'gradle' and 'yaml' are not correct. The spelling is OK but the CASE is not. I fixed these in PR #30397. With this fix you only get one Vale warning for  'gradle' and 'yaml' .